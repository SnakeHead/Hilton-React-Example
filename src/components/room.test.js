import React from 'react';
import Room from './room';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Room/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
