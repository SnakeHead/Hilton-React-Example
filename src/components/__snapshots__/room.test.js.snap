// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="roomPanel"
>
  <span
    className="roomTitle"
  >
    Room 
  </span>
  <div
    className="selectorPanel"
  >
    <form>
      <div
        className="roomSelect"
      >
        <label
          htmlFor="roomAdult"
        >
          Adults
          <br />
          (18+)
        </label>
        <select
          aria-label="roomAdults"
          disabled={true}
          name="roomAdult"
          onChange={[Function]}
        >
          <option>
            1
          </option>
          <option>
            2
          </option>
        </select>
      </div>
      <div
        className="roomSelect"
      >
        <label
          htmlFor="roomChild"
        >
          Children
          <br />
          (0-17)
        </label>
        <select
          aria-label="roomChildren"
          disabled={true}
          name="roomChild"
          onChange={[Function]}
        >
          <option>
            0
          </option>
          <option>
            1
          </option>
          <option>
            2
          </option>
        </select>
      </div>
    </form>
  </div>
</div>
`;
